datasets/ph/most_wanted/crawler.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    data = dict(row)
+    full_name = data.pop("name", None)
+    offense = data.pop("offense", None)
+    case_number = data.pop("case number", None)
+    source = data.pop("source", None)
+    reward = data.pop("reward", None)
+
+    entity = context.make("Person")
+    entity.id = context.make_id(full_name, case_number, reward)
+    entity.add("name", full_name)
+    entity.add("sourceUrl", source)
+    entity.add("topics", "wanted")
+
+    sanction = h.make_sanction(context, entity)
+    sanction.add("reason", offense)
+    sanction.add("program", "Most Wanted")
```

Comment:
I don't think a sanction entity is that helpful here. We mainly add it when there's data that really justifies adding it, like a sanction start/end date or an (official) program.

In this case the offense can go into the person:notes property along with the case numbers.